tests/src/Unit/OgMembershipTest.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace Drupal\Tests\og\Unit;
+
+use Drupal\Core\DependencyInjection\ContainerBuilder;
+use Drupal\og\Entity\OgMembership;
+use Drupal\og\Entity\OgRole;
+use Drupal\og\OgRoleInterface;
+use Drupal\Tests\UnitTestCase;
+
+/**
+ * Tests the OgMembership entity.
+ *
+ * @group og
+ * @coversDefaultClass \Drupal\og\Entity\OgMembership
+ */
+class OgMembershipTest extends UnitTestCase {
+
+  /**
+   * OgRole entity.
+   *
+   * @var \Drupal\og\Entity\OgRole|\Prophecy\Prophecy\ObjectProphecy
+   */
+  protected $ogRole;
+
+
+  /**
+   * {@inheritdoc}
+   */
+  public function setUp() {
+    $this->ogRole = $this->prophesize(OgRole::class);
+
+    // Set the container for the string translation service.
```

Comment:
I don't see the string translation service here but the "entity_type.repository" should be there.